manifests/controllerconfig.crd.yaml

```diff
@@ -961,6 +965,7 @@ spec:
             - ipFamilies
             - kubeAPIServerServingCAData
             - osImageURL
+            - oscontainer
```

Comment:
Any reason this isn't camel case like the fields around it?

Reply:
I struggle with camel casing and the two-letter `os` abbreviation.  I think you're right i n the existing style this should be `osContainer`.  But, perhaps it's better to bite the bullet and make it `operatingSystemContainer`?

Reply:
Yeah it's kinda weird. I think my personal preference would be `osContainer`, but :shrug: I guess we're going to drop `osImageURL` but that would also be consistent

Reply:
OK I went for maximum verbosity in `baseOperatingSystemContainer` because in code it's not actually referred to often.  Informally, we can just type "oscontainer" e.g. and not worry about camel casing.